Sources/ForageSDK/Foundation/Network/Utils/Environment.swift

```diff
@@ -7,7 +7,42 @@
 
 import Foundation
 
-
 internal func isUnitTesting() -> Bool {
     return ProcessInfo.processInfo.environment["XCTestConfigurationFilePath"] != nil
 }
+
+public enum Environment: String {
+    case dev = "dev"
+    case staging = "staging"
+    case sandbox = "sandbox"
+    case cert = "cert"
+    case prod = "prod"
+    
+    /// Returns the corresponding hostname for the environment.
+    public var hostname: String {
```

Comment:
Do you have an opinion about just organizing all static environment-affiliated data as a computed property under Envrionment?

For example, above you have:
```swift
private enum LDMobileKey: String {
    case sandbox = "mob-22024b85-05b7-4e24-b290-a071310dfc3d"
    case cert = "mob-d2261a08-784b-4300-a45f-ce0e46324d66"
    case prod = "mob-5c3dfa7a-fa6d-4cdf-93e8-d28ef8080696"
    case staging = "mob-a9903698-759b-48e2-86e1-c551e2b69118"
    case dev = "mob-03e025cb-5b4e-4d97-8685-39a22316d601"
}

private func getLDMobileKey(_ environment: Environment) -> LDMobileKey {
        switch environment {
        case .sandbox: return .sandbox
        case .cert: return .cert
        case .prod: return .prod
        case .staging: return .staging
        case .dev: return .dev
        }
    }
```

what are your thoughts on this:
```swift
public enum Environment: String {
    case dev = "dev"
    case staging = "staging"
    case sandbox = "sandbox"
    case cert = "cert"
    case prod = "prod"

    /// Returns the corresponding hostname for the environment.
    public var hostname: String {
        switch self {
        case .dev: return "api.dev.joinforage.app"
        case .staging: return "api.staging.joinforage.app"
        case .sandbox: return "api.sandbox.joinforage.app"
        case .cert: return "api.cert.joinforage.app"
        case .prod: return "api.joinforage.app"
        }
    }
    
    /// Returns the corresponding LDMobileKey value for the environment.
    public var ldMobileKeyValue: String {
        switch self {
        case .dev: return "mob-03e025cb-5b4e-4d97-8685-39a22316d601"
        case .staging: return "mob-a9903698-759b-48e2-86e1-c551e2b69118"
        case .sandbox: return "mob-22024b85-05b7-4e24-b290-a071310dfc3d"
        case .cert: return "mob-d2261a08-784b-4300-a45f-ce0e46324d66"
        case .prod: return "mob-5c3dfa7a-fa6d-4cdf-93e8-d28ef8080696"
        }
    }
}

```

Reply:
oh interesting, I'll create a ticket to scope this out